tsdb/head.go

```diff
@@ -1211,6 +1341,9 @@ func (h *Head) compactable() bool {
 
 // Close flushes the WAL and closes the head.
 func (h *Head) Close() error {
+	if err := h.chunkReadWriter.Close(); err != nil {
```

Comment:
I suspect you need to close the wal even if this fails.

Reply:
Yes